zio-http/shared/src/main/scala/zio/http/endpoint/AuthType.scala

```diff
@@ -15,6 +15,7 @@ sealed trait AuthType { self =>
       .asInstanceOf[
         AuthType { type ClientRequirement = ClientReq },
       ]
+
```

Comment:
Please add here methods to add and get scopes. In `Endpoint` there should be just proxies to these.